Documentation/direct-tools.md

```diff
@@ -16,7 +16,7 @@ If your pod dies, your mount will die with it.
 After you have created a pool as described in the [Block Storage](ceph-block.md) topic, you can create a block image and mount it directly in a pod.
 This example will show how the Ceph rbd volume can be mounted in the toolbox pod.
 
-After you have started and connected to the [Rook toolbox](ceph-toolbox.md), proceed with the following commands in the toolbox.
+After you have started and connected to the [Rook toolbox](direct-mount.md), proceed with the following commands in the toolbox.
```

Comment:
You'll need to remove the link to the toolbox and instead give instructions to start the direct-mount.yaml

Reply:
I have changed the occurrences to `direct-mount.yaml` and placed it in the same place where `toolbox.yaml` is being placed.